.github/workflows/promote-to-prod.yml

```diff
@@ -0,0 +1,81 @@
+name: Promote to Amazon ECS prod environment
+
+on: 
```

Comment:
I'm not sure we're quite ready for this code yet. A lot can change from here to February/March. Can we separate this workflow into a separate PR?

Reply:
removed entire file from PR